pudb/debugger.py

```diff
@@ -1595,6 +1595,7 @@ def add_cmdline_content(s, attr):
             self.cmdline_list.set_focus_valign("bottom")
             self.cmdline_list.set_focus(len(self.cmdline_contents) - 1,
                     coming_from="above")
+        self.add_cmdline_content = add_cmdline_content
```

Comment:
I think it would be useful if `add_cmdline_content` could force-show the shell (at least until the next time it's focused in the case of auto-hiding) to ensure the output is discoverable.